install.sh

```diff
@@ -11,52 +11,61 @@
 TMP_DIR="/tmp/install-fossa-cli"
 
 function cleanup {
-  echo rm -rf $TMP_DIR > /dev/null
+  rm -rf $TMP_DIR > /dev/null
 }
+trap cleanup EXIT
 
 function fail {
-  cleanup
   msg=$1
   echo "============"
   echo "Error: $msg" 1>&2
   exit 1
 }
 
+# This function will ask for root privileges before executing a command
+# The goal is to allow the used to run this script as a normal user and
+# to ash authorizations as needed
```

Comment:
Some minor corrections:
- "allow the used to run" --> "allow the user to run"
- "to ash authorizations as needed" --> "to be asked for authorizations as needed"

Reply:
fixed in b67280f :)